scalafix-tests/unit/src/test/scala/scalafix/tests/cli/ScalafixImplSuite.scala

```diff
@@ -97,6 +99,47 @@ class ScalafixImplSuite extends AnyFunSuite with DiffAssertions {
     assert(e.isPresent)
     assert(e.get().getMessage.contains("-Ywarn-unused"))
   }
+  test("configure scalafix path") {
+    val api = i.Scalafix.classloadInstance(this.getClass.getClassLoader)
+
+    val charset = StandardCharsets.US_ASCII
+    val cwd = StringFS
+      .string2dir(
+        """|/src/Semicolon.scala
+           |
+           |object Semicolon {
+           |  def main { println(42) }
+           |}
+           |/.scalafix.conf
+           |rules = ["DisableSyntax"]
+      """.stripMargin,
+        charset
+      )
+    val args = api
+      .newArguments()
+      .withConfig(Optional.empty())
+      .withWorkingDirectory(cwd.toNIO)
+    args.validate()
+    assert(
+      args.rulesThatWillRun().asScala.toList.map(_.toString) == List(
```

Comment:
do you really need to fallback on a `toString` for assertion? (same comment for the second assertion)